### ISSUE TITLE
AWS Aroura read replica support from ebean side

### ISSUE THREAD

**Issue by riteshmaurya** [ebean-orm/ebean#1994]
GITHUB ISSUES ARE STRICTLY CONTROLLED FOR THIS PROJECT.

Refer to http://ebean-orm.github.io/support for the policies controlling the use of github issues.
Please post issues to the Ebean group https://groups.google.com/forum/#!forum/ebean first.

ebean should automatically direct read request to reader instance of aws aurora cluster and write request to writer instance

developer should have an option of enable/disable using properties file.

**Comment by rbygrave** [comment]
Yes.  This is really a documentation issue as Ebean already has support for this via configuring a read only DataSource.  

```java
// jdbc datasource that uses the aws aurora `-ro` endpoint
DataSource readonlyDataSource = ... 

databaseConfig.setReadonlyDataSource(...)
```
Ebean will automatically send query requests that are not in a `@Transactional` and not a `for update` to the read only DataSource.

**Outstanding task:**
- Add documentation for this

**Comment by rbygrave** [comment]
Docs are at: https://ebean.io/docs/read-replicas/#aurora

closing.
